CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## [4.0.8](https://github.com/vuejs/vue-router-next/compare/v4.0.6...v4.0.8) (2021-05-13)
+
+This release is a correct tag and doesn't contain any extra changes.
+
+## [4.0.7](https://github.com/vuejs/vue-router-next/compare/v4.0.6...v4.0.7) (2021-05-13)
+
+### Bug Fixes
+
+- **$route:** Make $route have an enumerable type ([#912](https://github.com/vuejs/vue-router-next/issues/912)) ([d90520e](https://github.com/vuejs/vue-router-next/commit/d90520eb36ca310f52b22caeb88fd8cb06dca04a))
```

Comment:
These changes should not be here. Could you rebase from `master`?

Reply:
yes，There is a conflict between master and issues-929.I rebase the master.

I don't understand very well, can you teach me what is the correct way? Thank you.

Reply:
Can I close this PR and then pull a branch to raise the PR?

Reply:
Sure, feel free to open a new PR.

Reply:
I have eliminated irrelevant commits， and forced to push to the issues-929 branch.  the PR doesn't contain extra code changes. 

Do you think it is okay? thanks.